config/Config-kernel.in

```diff
@@ -1184,9 +1195,56 @@ config KERNEL_SECURITY_SELINUX_SID2STR_CACHE_SIZE
 	depends on KERNEL_SECURITY_SELINUX
 	default 256
 
+config KERNEL_SECURITY_APPARMOR
+	bool "AppArmor Support"
+	select KERNEL_AUDIT
+	select KERNEL_SECURITY
+	select KERNEL_SECURITY_FS
+	select KERNEL_SECURITY_PATH
+	select KERNEL_SECURITY_NETWORK
+
+config KERNEL_DEFAULT_SECURITY_APPARMOR
+	bool "Select AppArmor as default security module"
+
+config KERNEL_SECURITY_APPARMOR_BOOTPARAM_VALUE
+	int
+	default 0
+	help
+	  This option sets the default value for the kernel parameter
+	  'apparmor', which allows AppArmor to be enabled or disabled
+	  at boot. If this option is set to 0 (zero), the AppArmor
+	  kernel parameter will default to 0, disabling AppArmor at
+	  boot. If this option is set to 1 (one), the AppArmor
+	  kernel parameter will default to 1, enabling AppArmor at
+	  boot.
+
+	  AppArmor and SELinux cannot be used simultaneously, choose
+	  only one of them as default.
+	depends on KERNEL_SECURITY_APPARMOR
+
+config KERNEL_SECURITY_APPARMOR_HASH
+	bool "Enable introspection of sha1 hashes for loaded profiles"
+	depends on KERNEL_SECURITY_APPARMOR
+
+config KERNEL_SECURITY_APPARMOR_HASH_DEFAULT
+	bool "Enable policy hash introspection by default"
+	depends on KERNEL_SECURITY_APPARMOR_HASH
+
+config KERNEL_SECURITY_APPARMOR_DEBUG
+	bool "Build AppArmor with debug code"
+	depends on KERNEL_SECURITY_APPARMOR
+
+config KERNEL_SECURITY_APPARMOR_DEBUG_ASSERTS
+	bool "Build AppArmor with debugging asserts"
+	depends on KERNEL_SECURITY_APPARMOR_DEBUG
+
+config KERNEL_SECURITY_APPARMOR_DEBUG_MESSAGES
+	bool "AppArmor debug messages enabled by default"
+	depends on KERNEL_SECURITY_APPARMOR_DEBUG
+
 config KERNEL_LSM
 	string
-	default "lockdown,yama,loadpin,safesetid,integrity,selinux"
+	default "lockdown,yama,apparmor,loadpin,safesetid,integrity,selinux"
```

Comment:
This shall not be set by default...

Reply:
@osikarirauta Do you know what happens if it is not available otherwise?

Reply:
I'm wondering if we can just delete this line entirely. Since you won't have both installed/enabled we can just stick to the default with comes with the kernel.